modules/installation-osp-default-deployment.adoc

```diff
@@ -2,19 +2,24 @@
 //
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
 
 ifeval::["{context}" == "installing-openstack-installer-custom"]
 :osp-custom:
 endif::[]
 
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-user-infra:
+endif::[]
+
 [id="installation-osp-default-deployment_{context}"]
 = Resource guidelines for installing {product-title} on OpenStack
 
-Your quota must meet the following requirements to run the {product-title} installation program in {rh-openstack-first}.
+To support a {product-title} installation, your {rh-openstack-first} quota should meet the following requirements:
```

Comment:
Could you add a note stating that if an installation using `Kuryr SDN` is desired the cluster requirements should be checked at: " http://file.rdu.redhat.com/mbridges/022720/shiftstack-upi/installing/installing_openstack/installing-openstack-installer-kuryr.html ?

Reply:
We tend to want pages to be self-contained, so I'll think about this. Likely options are:

* Including Kuryr resource requirements here
* Creating a separate Kuryr user-provisioned infrastructure guide (as happened with the installation-provisioned infrastructure guides)

Reply:
The Kuryr flow differs enough from non-Kuryr flows that I think it warrants its own assembly. This is a bit of an IA headache, but it's the only option right now. 

Addressed in https://github.com/openshift/openshift-docs/pull/20091/commits/cf06170bef01760271dbc0e84df15a31e18a08ff